modules/clickhouse/src/main/java/org/testcontainers/containers/ClickHouseProvider.java

```diff
@@ -1,15 +1,18 @@
 package org.testcontainers.containers;
 
-import org.testcontainers.utility.DockerImageName;
-
 public class ClickHouseProvider extends JdbcDatabaseContainerProvider {
     @Override
     public boolean supports(String databaseType) {
-        return databaseType.equals(ClickHouseContainer.NAME);
+        return databaseType.equals(ClickHouseInit.JDBC_NAME_YANDEX) || databaseType.equals(ClickHouseInit.JDBC_NAME_MYSQL);
```

Comment:
Wouldn't this collide with the normal mysql JDBC URL if someone has both modules on the classpath? In this case, they would get either container.

Reply:
It would be interesting to see what happens in this case, but I think it's a good point. I wonder if we should:
(a) make the `ContainerDatabaseDriver` fail fast in this scenario, or 
(b) require a different string to be used, e.g. `tc:clickhousemysql`. This seems less nice though, as it breaks an implicit contract about use of `tc:` in URLs.

Reply:
While (a) sounds reasonable (and I wonder why we decided to return **any** matching container in case of collision) how would a user that has both dependencies on their classpath (for a valid reason) deal with this? Our tests would also fail, or?